src/content/development/website/_index.en.md

```diff
@@ -21,6 +21,8 @@ submitting you can:
     ```
 
 3. An instance of the website is now running locally on your machine and is accessible at <http://localhost:1313>.
+    (Instead of running just `make server` above,
```

Comment:
This needs one space removed: the four spaces at the start of the line mark a code block, which is what is tripping up the linter.

```suggestion
   (Instead of running just `make server` above,
```

Reply:
Thank you for pointing it out. It seems that just 4 spaces don't enough, so I've added one blank line above.